src/components/EmailForm/index.jsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from "react"
+import { Link } from "react-router-dom"
+
+function EmailForm(){
+    const [email,setEmail]=useState('')
+    const [subject,setSubject]=useState('')
+    const [message,setMessage]=useState('')
+    const handleSubmit=(e)=> {
+        e.preventDefault()
+        console.log(email)
+        console.log(subject)
+        console.log(message)
+        getData()
+    }
+  function getData (){
+    fetch('https://email-client-api.dev.io-academy.uk/emails', {
+  method: 'POST',
+  headers: {
+    "content-type": "application/json"
+  },
+  body:JSON.stringify({
+    name: "username",
+     email:email,
+      subject:subject,
+      body:message
+      })
+})
+  .then(res => res.json())
+  .then(data => {
+    console.log(data)
+  })
+  }
+ return(
+        <form onSubmit={handleSubmit}
+            className="flex flex-col py-4 px-8 gap-4">
+            <label htmlFor="email" />
+            <input
+            type="email"
+            id="email"
+            value={email}
+            onChange={(e)=> setEmail(e.target.value)}
+            placeholder="To"
+            className="border-2 rounded border-gray-200 p-2"
+            />
+            <label htmlFor="subject" />
+            <input
+            type="text"
+            id="subject"
+            value={subject}
+            onChange={(e)=> setSubject(e.target.value)}
+            placeholder="Subject"
+            className="border-2 rounded border-gray-200 p-2"
+            />
```

Comment:
These two inputs are almost identicle, anyway you can make a re-usable component instead?